lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/handler/DrivingPermitHandler.java

```diff
@@ -180,7 +184,11 @@ public APIGatewayProxyResponseEvent handleRequest(
             dataStore.create(documentCheckResultItem);
             LOGGER.info("document check results saved.");
 
-            return ApiGatewayResponseGenerator.proxyJsonResponse(HttpStatusCode.OK, result);
+            // TODO: Create parameter in store
+            Attempts retry = new Attempts();
```

Comment:
I think this logically lives in DL rather than common also can rename it as its a bit confusing. Something like DocumentVerificationResponse